docs/admin/install.rst

```diff
@@ -0,0 +1,243 @@
+
+Apply updates to system templates
+---------------------------------
+Before installing SecureDrop Workstation, you must set up network and Tor access, then update the system VMs:
+
+- After logging in, use the network manager widget in the upper-right panel to configure your network connection.
+
+- Next, configure Tor by selecting the Qubes menu (the **Q** icon in the upper left corner) and selecting **Service: sys-whonix > sys-whonix: Anon Connection Wizard**. In most cases, choosing the default**Connect** option is best. Click **Next**, then **Next** again. Then, if Tor connects successfully, click **Finish**. If Tor fails to connect, make sure your network conection is up and does not filter Tor connections, then try again.
+
+  .. note:: If Tor connections are blocked on your network, you may need to configure Tor to use bridges in order to get a connection. For more information, see the `Anon Connection Wizard <https://www.whonix.org/wiki/Anon_Connection_Wizard>`_ documentation.
+
+
+- Open an administration terminal by selecting **Q > Terminal Emulator**, and enter the following command:
+
+  .. code-block:: sh
+
+   sudo qubes-dom0-update --enablerepo=qubes-dom0-current-testing
+
+  This will update ``dom0``, the administration VM, with some required fixes to properly update other VMs. When prompted, type **Y** to apply the updates.
+
+
+- Once the ``dom0`` update is complete, select **Q > System Tools > Qubes Update** to update other VMs. in the ``[Dom0] Qubes Updater`` window, first check ``Enable updates for qubes without known available updates``, then check all entries in the list above. Then, click **Next**. The system's VMs will be updated sequentially - this may take some time. When the updates are complete, click **Finish**.
+
+
+Copy the submission key
+-----------------------
+In order to decrypt submissions, your SecureDrop Workstation will need a copy of the secret key from your SecureDrop instance's SVS. To protect this key and preserve the air gap, you will need to connect the SVS USB to a Qubes VM with no network access, and copy it from there to ``dom0``:
+
+- First, use the network manager widget in the upper right panel to disable your network connection. These instructions refer to the ``vault`` VM, which has no network access by default, but if the SVS USB is attached to another VM by mistake, this will offer some protection against exfiltration.
+
+- Next, choose **Q > Domain: vault > vault: Files** to open the file manager in the ``vault`` VM.
+
+- Connect the SVS USB to a USB port on the Qubes computer, then use the devices widget in the upper right panel to attach it to the ``vault`` VM. There will be 3 listings for the USB in the widget: one for the base USB, one for the Tails partition on the USB, labeled ``Tails``, and a 3rd unlabeled listing, for the persistent volume. Choose the third listing.
```

Comment:
This would be better handled in a generic qubes intro, or via screenshots imo.

Reply:
I think a screenshot is needed here, especially because the widget lists USB devices both as block devices and as physical devices, which can be very confusing.

Reply:
I've taken two screenshots; will upload shortly.